src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days < 3) {
+    result = days * 40;
+  } else {
```

Comment:
```suggestion
  } else if(days <= 6) {
```
Можно проверку тут сразу делать